tests/integration/Event_Cleaner_Scheduler_Test.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * Test that only past events are purged
+ */
+
+use Tribe__Events__Event_Cleaner_Scheduler as Scheduler;
+
+
+class Event_Cleaner_Scheduler_Test extends \Codeception\TestCase\WPTestCase {
+
+	protected $settings;
+
+	public function event_settings( $event_date ) {
+		$this->settings = array(
+			'post_author'           => 1,
+			'post_title'            => 'Test event',
+			'post_content'          => 'This is event content!',
+			'post_status'           => 'publish',
+			'EventAllDay'           => false,
+			'EventHideFromUpcoming' => true,
+			'EventOrganizerID'      => 1,
+			'EventVenueID'          => 1,
+			'EventShowMapLink'      => true,
+			'EventShowMap'          => true,
+			'EventStartDate'        => $event_date,
+			'EventEndDate'          => $event_date,
+			'EventStartHour'        => '01',
+			'EventStartMinute'      => '15',
+			'EventStartMeridian'    => 'am',
+			'EventEndHour'          => '03',
+			'EventEndMinute'        => '25',
+			'EventEndMeridian'      => 'pm',
+		);
+
+		return $this->settings;
+	}
+
+	/**
+	 * Check to make sure that past events are being correctly selected.
+	 */
+	public function test_tribe_select_only_past_events() {
+		$past_event_date     = '2018-01-01';
+		$upcoming_event_date = date( 'Y-m-d' );
+		$past_event          = tribe_create_event( $this->event_settings( $past_event_date ) );
+		$upcoming_event      = tribe_create_event( $this->event_settings( $upcoming_event_date ) );
```

Comment:
total nitpick, but since `$past_event_date` and `$upcoming_event_date` aren't being re-used anywhere in this file, you could just get rid of those vars and pass the literal dates to your `tribe_create_event` calls, like:

```
$past_even      = tribe_create_event( $this->event_settings( '2018-01-01' ) );
$upcoming_event = tribe_create_event( $this->event_settings( date( 'Y-m-d' ) ) ); 
```

this **not** a blocker for merging tho, so will not tag this `enhance` or anything—just wanted to call it out in comments.

nice work!